test/Tests/src/Semantic/Instrumentor_Spec.enso

```diff
@@ -44,6 +45,38 @@ spec =
             # no more instrumenting after finalize
             b.to_vector.length . should_equal 1
 
+        Test.specify "access local variables " <|
+            b = Vector.new_builder
+
+            collect uuid:Text ~result =
```

Comment:
The `~result` must be a suspended argument, so it only gets evaluated when `uuid == a_plus_b_uuid` - otherwise it the framework may evaluate `[a, b, a-b]` even in situations when the variables `a` and `b` aren't specified.